helm/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: cloudzero-agent
 description: A chart for using Prometheus in agent mode to send cluster metrics to the CloudZero platform.
 type: application
 version: 1.1.0-dev
-kubeVersion: ">= 1.23.0"
+kubeVersion: ">= 1.23.0-0"
```

Comment:
@spkane my tests failed because my server version is `v1.30.10-eks-bc803b4`, which I think it interpreted as a pre-release version? do you think this change is acceptable?